airbyte-cdk/java/airbyte-cdk/README.md

```diff
@@ -156,6 +156,7 @@ MavenLocal debugging steps:
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                                                        |
 |:--------|:-----------|:-----------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0.1.8   | 2023-10-11 | [\#31322](https://github.com/airbytehq/airbyte/pull/31322) | Cap log line length to 32KB to prevent loss of records                                                                                                          |
```

Comment:
Should this be updated to the PR that actually introduced the change?

Reply:
Ideally it would have been the same one. but because of the time that passed I forgot this was in the CDK.
I referred to this one in case someone needs to reconstruct the chain of events and mentioned the issue in description.